esmvaltool/cmorizers/data/cmor_config/ESACCI-LANDCOVER.yml

```
@@ -0,0 +1,56 @@
+attributes:
+  project_id: 'OBS'
+  dataset_id: 'ESACCI-LANDCOVER'
+  tier: 2
+  modeling_realm: sat
+  institution: ''
+  reference: 'esacci-landcover'
+  source: 'ftp://anon-ftp.ceda.ac.uk/neodc/esacci/land_cover/data/pft'
+  title: 'ESA CCI Land Cover'
+  version: 'v2.0.8'
+  comment: ''
```

```suggestion
  comment: ''
filename: ESACCI-LC-L4-PFT-Map-300m-P1Y-{year}-v2.0.8.nc
```

Since this is always the same